### PR TITLE
Update Openshift SCC docs link

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**
# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [x] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Review by mateusoliveira43 — APPROVED** [review]

**Review by kaovilai — APPROVED** [review]

**Review by blackpiglet — DISMISSED** [review]

File: site/content/docs/main/file-system-backup.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ To mount the correct hostpath to pods volumes, run the node-agent pod in `privil
 
 If node-agent is not running in a privileged mode, it will not be able to access pods volumes within the mounted 
 hostpath directory because of the default enforced SELinux mode configured in the host system level. You can 
-[create a custom SCC](https://docs.openshift.com/container-platform/3.11/admin_guide/manage_scc.html) to relax the 
+[create a custom SCC](https://docs.openshift.com/container-platform/4.16/authentication/managing-security-context-constraints.html) to relax the
```
Comment (mpryc) [inline site/content/docs/main/file-system-backup.md:141]: How about:
https://docs.openshift.com/container-platform/latest/authentication/managing-security-context-constraints.html

Comment (kaovilai) [reply site/content/docs/main/file-system-backup.md:141]: The path might break in the future. I've seen couple docs where bumping version result in 404.

Comment (shubham-pampattiwar) [reply site/content/docs/main/file-system-backup.md]: Done, updated.

**Review by mpryc — APPROVED** [review]
/lgtm

**Review by kaovilai — APPROVED** [review]

**Review by Lyndon-Li — APPROVED** [review]

**Review by blackpiglet — APPROVED** [review]

**Pull request merged**